### ISSUE TITLE
Rate-limit support

### ISSUE THREAD

**Issue by phorcys420** [Ylianst/MeshCentral#4514]
**Is your feature request related to a problem? Please describe.**
I don't like bots being able to password guess easily.

**Describe the solution you'd like**
I would like to be able to define rate-limit rules.
e.g for ``5`` failed login attemps, the user can't try to login for a minute.

**Describe alternatives you've considered**
CrowdSec can be a nice combination to this, but definitely not an alternative.

**Comment by VINISHVV** [comment]
Both features are currently available in meshcentral. Please check advance configuration here [https://github.com/Ylianst/MeshCentral/blob/master/meshcentral-config-schema.json]() for crowdsec and IP ban

**Comment by Ylianst** [comment]
Exactly. Also, password guessing rate limiting has been present in MeshCentral for a long time. The default is 10 tries in 10 minutes. You can go in "My Server / Console" tab and type "badlogins" to see the current state.

![image](https://user-images.githubusercontent.com/1319013/188719033-848e243e-c73c-4bfc-aad9-f25447dae123.png)

![image](https://user-images.githubusercontent.com/1319013/188718725-0e696831-3b46-407e-95a5-2c1845f98f21.png)

**Comment by phorcys420** [comment]
Thanks ! I didn't know it was configurable !
Also I only tried with about 8 logins so I assumed it was not present.

I will check tomorrow but it should be good.

**Comment by Ylianst** [comment]
Thanks for responding. Yes, do test it and feel free to customize to your needs. Close this issue if it works for you.

**Comment by phorcys420** [comment]
Thanks ! it works, closing as resolved.

**Comment by phorcys420** [comment]
> Both features are currently available in meshcentral. Please check advance configuration [here](https://github.com/Ylianst/MeshCentral/blob/master/sample-config-advanced.json) for crowdsec and IP ban

Yes, I was unclear, I meant that CrowdSec on it's own is nice but that it wouldn't be enough without rate-limit.
